extensions/etcdsnapshot/etcdsnapshot.go

```diff
@@ -127,23 +75,35 @@ func CreateRKE1Snapshot(client *rancher.Client, clusterName string) error {
 			if snapshotObj.State != active {
 				return false, nil
 			}
+
+			snapshotTime, err := time.Parse(time.RFC3339, snapshot.Created)
+			if err != nil {
+				return false, err
+			}
+
+			// time.Parse doesn't include nanoseconds, but time.Now() does. Rounding up by 1 Second.
+			snapshotTime = snapshotTime.Add(time.Duration(oneSecond))
```

Comment:
Could you use the time.Second constant?

`snapshotTime = snapshotTime.Add(time.Duration(time.Second))`